### PR TITLE
Adding Olingo 4.3.0 library

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**

### CODE REVIEW

File: olingo-server/4.2.0wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                            com.fasterxml.aalto.stax;resolution:=optional,
+                            com.fasterxml.jackson.core;version="[2.4,3)",
+                            com.fasterxml.jackson.core.util;version="[2.4,3)",
+                            com.fasterxml.jackson.databind;version="[2.4,3)",
+                            com.fasterxml.jackson.databind.node;version="[2.4,3)",
+                            javax.servlet,
+                            javax.servlet.http,
+                            javax.xml.namespace,
+                            javax.xml.stream,
+                            javax.xml.stream.events,
```
Comment (Niranjan-K) [inline olingo-server/4.2.0wso2v1/pom.xml]: Please define the version range

File: olingo-server/4.2.0wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                            org.antlr.v4.runtime;resolution:=optional,
+                            org.antlr.v4.runtime.atn;resolution:=optional,
+                            org.antlr.v4.runtime.dfa;resolution:=optional,
+                            org.antlr.v4.runtime.misc;resolution:=optional,
+                            org.antlr.v4.runtime.tree;resolution:=optional,
```
Comment (Niranjan-K) [inline olingo-server/4.2.0wso2v1/pom.xml]: please mention the version range

File: olingo-server/4.2.0wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.apache.olingo.server.core.*; version="${olingo.server.export.version}",
+                            org.apache.olingo.commons.api.*; version="${olingo.server.export.version}",
+                            org.apache.olingo.server.api.*; version="${olingo.server.export.version}",
+                            org.apache.olingo.commons.core.*; version="${olingo.server.export.version}"
```
Comment (Niranjan-K) [inline olingo-server/4.2.0wso2v1/pom.xml]: export version should not have the wso2v* suffix.

**Comment by madhawa-gunasekara** [comment]
Please find the relevant jira https://wso2.org/jira/browse/DS-1220

File: olingo-server/4.3.0wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline olingo-server/4.3.0wso2v1/pom.xml]: remove snapshot repo

Comment (madhawa-gunasekara) [inline olingo-server/4.3.0wso2v1/pom.xml]: +1

will remove it

**Pull request merged**